src/2_RTB/VerbaleEsterno_101123/content.typ

```diff
@@ -0,0 +1,67 @@
+
+Inoltre, dato che i ruoli all'interno del team ruoteranno ad ogni Sprint, una durata media più estesa dovrebbe contribuire a consolidare le responsabilità connesse ai vari ruoli. La Proponente ha manifestato il suo accordo riguardo a questa modalità di lavoro e si è offerta di organizzare incontri periodici di Stato di Avanzamento Lavori (SAL) alla conclusione di ciascuno Sprint.
+
+A titolo di esempio, considerando che il primo Sprint è iniziato oggi, il prossimo incontro SAL sarà fissato per venerdì 24 novembre, a distanza di esattamente due settimane.
+
+=== Strumentazione di lavoro
+Nonostante la proposta iniziale di utilizzare Discord avanzata dalla Proponente, durante l'incontro odierno si è optato per l'adozione della piattaforma Element, basata sul protocollo Matrix, come principale strumento di comunicazione. La Proponente ha espresso la volontà di utilizzare Element rispetto ad altre opzioni come Discord o Slack, motivando tale scelta con considerazioni legate alla sicurezza e alla facilità di comunicazione asincrona.
+
+=== Stack tecnologico conosciuto dal team
+La Proponente ha manifestato l'interesse di approfondire la conoscenza dello stack tecnologico del team al fine di definire un carico di lavoro adeguato per il completamento del primo Sprint. Durante la discussione, il team ha comunicato alla Proponente la decisione di adottare Python come linguaggio principale per gli script di simulazione dei sensori. Questa scelta è stata motivata dalla preferenza per un linguaggio meno verboso e di apprendimento più agevole rispetto ad altri come Go o Rust, con l'obiettivo di garantire un livello uniforme di padronanza all'interno del team. La Proponente ha approvato questa decisione, sottolineando la necessità di evitare l'introduzione di nuove tecnologie che potrebbero complicare l'apprendimento nel breve periodo e influire negativamente sull'avanzamento del progetto.
+
+=== Obiettivi dello Sprint
+Dopo aver comunicato alla Proponente l'intenzione del team di effettuare la prima revisione con il Committente entro il 18 dicembre, sono stati concordati gli obiettivi chiave per il primo Sprint:
+
+- *Documentazione RTB:* Il team si impegna a continuare il lavoro sulla documentazione relativa al _Requirements and Technology Baseline_ (RTB), ponendo particolare attenzione a Norme di Progetto e Analisi dei Requisiti;
+
+- *Implementazione di uno script di simulazione:* Sarà sviluppato uno script in grado di simulare l'attività di uno o più sensori. Lo script emetterà uno stream di dati strutturato, ad esempio in formato JSON, inviandolo a Kafka;
+
+- *Utilizzo di Docker Composed:* Il team adotterà Docker Composed per creare un ambiente containerizzato di base. Questo ambiente conterrà le immagini dei vari componenti dell'architettura, facilitando la gestione e la distribuzione delle risorse;
```

Comment:
Sostituirei con: 
Ambiente Contenerizzato: il team si impegna realizzare un script Docker compose che configuri i singoli servizi dell'architettura come conteiner e che definisca le loro connessioni al fine di permettere una più facile gestione e distribuzione delle risorse.

Inoltre personalmente aggiungerei (ma valutate voi se è adeguato) che questo script non ha proprio niente di definitivo in quanto tecnicamente anche in mancanza di un'analisi dei requistiti, non siamo neanche certi che le tecnologie (mi sembra solo consigliate, ma non obbligatorie), siano le più adeguate. 
Per come l'ho vista io è più una cosa per farci mettere le mani nell'impasto di docker e dei conteiner.

Reply:
Sono d'accordo ma non credo serva menzionarlo nel verbale, se ci pensi vale lo stesso anche per lo script di simulazione dei sensori che verra' cambiato in modo sostanziale. Di fatto e' un'obiettivo di questo primo Sprint quindi mi atterrei alla sostituzione che hai suggerito. Sostituito.

Reply:
perfetto